### ISSUE TITLE
Some features missing

### ISSUE THREAD

**Issue by paulirish** [Niek/chrome-features#1]
As you (i think) know... i maintain https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md

I've been updating it again recently.. Found some new common feature flags but they're not in your list..

```
AutofillServerCommunication
GlobalMediaControls
HeavyAdPrivacyMitigations
ImprovedCookieControls
InterestFeedContentSuggestions
```

You can peek at the first column of https://docs.google.com/spreadsheets/d/1n-vw_PCPS45jX3Jt9jQaAhFqBY6Ge1vWF_Pa0k7dCk4/edit#gid=1265672696 for several more.

Thanks for making this handy webapp :)

**Comment by Niek** [comment]
Thanks!

Based on these, I need to parse these extra files:
* [`autofill_features.cc`](https://source.chromium.org/chromium/chromium/src/+/main:components/autofill/core/common/autofill_features.cc)
* [`media_switches.cc`](https://source.chromium.org/chromium/chromium/src/+/main:media/base/media_switches.cc)
* [`heavy_ad_features.cc`](https://source.chromium.org/chromium/chromium/src/+/main:components/heavy_ad_intervention/heavy_ad_features.cc)
* [`features.cc`](https://source.chromium.org/chromium/chromium/src/+/main:components/content_settings/core/common/features.cc)
* [`feed_feature_list.cc`](https://source.chromium.org/chromium/chromium/src/+/main:components/feed/feed_feature_list.cc)

I'm pretty sure there are others though that I'm missing... let me know if you find more.

**Comment by paulirish** [comment]
Yeah my previous assumption is that its every file where you can find `const base::Feature`. It seems like "feature" in the filename is a convention but not a requirement.

**Comment by Niek** [comment]
The latest `main` branch has a lot of changes for the flags, now you can easily grep for `^BASE_FEATURE\(` it seems. I'll do some more tests to see if I can build a full dump.

URL: https://source.chromium.org/search?q=%5EBASE_FEATURE%5C(&ss=chromium%2Fchromium%2Fsrc

**Comment by Niek** [comment]
Added in `main` via 8013f10.

This now parses the extra feature files mentioned here, generalizes `BASE_FEATURE` extraction across namespaces, and expands preference coverage with additional Chromium pref-name sources. The generated page now includes the missing feature families like Autofill, media controls, heavy-ad, content settings, and feed features.

Closing this as fixed.
